simulator/risc_v/riscv_decoder.h

```diff
@@ -16,8 +16,17 @@
 enum class Reg : uint8
 {
     RS1, RS2, RD,
+    RS1_, RS2_, RD_,
```

Comment:
What does "_" mean here? We use self-descriptive names.

Reply:
There are registers " rs1 ' ",   " rs2 ' ", " rd ' " which are related to 8 most popular registers. So these registers are just 3 bits long. Could you please suggest me how I should call them?

Reply:
`rs1_3_bits_long` or `rs1_compressed`